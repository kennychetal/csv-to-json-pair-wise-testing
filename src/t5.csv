letter,letter2,symbol,
j,a,@,
m,x,$,
a,y,%,