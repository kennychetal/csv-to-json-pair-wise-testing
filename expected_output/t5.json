[
    {
        "symbol": "@", 
        "letter2": "a", 
        "letter": "j"
    }, 
    {
        "symbol": "$", 
        "letter2": "x", 
        "letter": "m"
    }, 
    {
        "symbol": "%", 
        "letter2": "y", 
        "letter": "a"
    }
]